chart/templates/post-delete-hook-job.yaml

```
@@ -0,0 +1,47 @@
+{{- if .Values.postDelete.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Values.postDelete.name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "rancher.labels" . | indent 4 }}
+  annotations:
+    "helm.sh/hook": post-delete
+    "helm.sh/hook-weight": "3"
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+spec:
+  backoffLimit: 3
+  template:
+    metadata:
+      name: {{ .Values.postDelete.name }}
+      labels:
+        heritage: {{ .Release.Service }}
+        release: {{ .Release.Name }}
+    spec:
+      serviceAccountName: {{ .Values.postDelete.name }}
+      restartPolicy: OnFailure
+      containers:
+        - name: {{ .Values.postDelete.name }}
+          image: "{{ .Values.systemDefaultRegistry }}{{ .Values.postDelete.image.repository }}:{{ .Values.postDelete.image.tag }}"
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            runAsUser: 0
+          command:
+            - /script/post-delete-hook.sh
+          volumeMounts:
+            - mountPath: /script
```

In case we add more scripts in the future. Will also need to change the command to `/scripts/post-delete-hook.sh`

```suggestion
            - mountPath: /scripts
```

---

fixed.